examples/rstudio/.saturn/saturn.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "example-rstudio",
+    "image_uri": "saturncloud/saturn-rstudio:2021.10.26",
```

Comment:
Not an issue now, but this should get bumped to `2021.11.0`. Want me to PR?

Reply:
It should get that bump on the release branch though right?

Reply:
Oh, right